### ISSUE TITLE
[Feature request] Make it possible to start a live stream from the beginning while it's still streaming.

### ISSUE THREAD

**Issue by emitho** [iv-org/invidious#3416]
**Is your feature request related to a problem? Please describe.**

**Describe the solution you'd like**

**Describe alternatives you've considered**

**Additional context**

Currently it is not possible to go back to the beginning of a video that is still streaming live on YouTube.

**Comment by unixfox** [comment]
Is this even possible on Youtube interface itself?

**Comment by emitho** [comment]
Yes! I follow a youtuber closely who exclusively uploads livestreams. He is the only channel I have problems with on Invidious (it also takes a couple of hours before his videos appear in the subscription feed after live stream is done). When I used youtube I could scrub back and forth in the live stream while it was going on.Enviado desde mi iPhoneEl 23 nov 2022, a la(s) 7:24, Émilien Devos ***@***.***> escribió:﻿
Is this even possible on Youtube interface itself?

**Comment by TheShanMan** [comment]
Please, please, please! I watch live streams regularly in not quite real time (i.e. live stream is still going but I join late). I hate having to use youtube.com for live streams because invidious doesn't give me a slider bar to move back to the beginning of the stream.

**Comment by ekipan** [comment]
@unixfox 
I think it depends on the live stream. I've seen some that allow me to seek backwards and some that don't. Not sure if that's something the streamer has control over or not. Here's one that currently does at time of posting: [Chiptune Radio on Youtube](https://www.youtube.com/watch?v=IlCoKe_F2G8)

**Comment by TheShanMan** [comment]
Maybe that's true, and if so, this request would obviously only be for streams that do support seeking backwards. The streams I care about allow me to seek backwards on youtube.com but not invidious, and that's what I hope to see invidious support. Thanks for chipping in with your observations.

**Comment by Giger22** [comment]
Any update?
